package.json

```diff
@@ -12,37 +12,50 @@
     "react-helmet": "^5.2.0",
     "react-router": "^4.1.2",
     "react-router-dom": "^4.1.2",
-    "react-scripts": "1.0.11",
     "semantic-ui-css": "2.2.12",
     "semantic-ui-react": "^0.75.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
-    "coverage": "react-scripts test --coverage",
+    "test": "cross-env CI=true react-scripts test",
+    "test:coverage": "cross-env CI=true react-scripts test --coverage",
     "eject": "react-scripts eject",
-    "deploy": "npm run build&&gh-pages -d build",
-    "generateSNAPTests": "./node_modules/.bin/babel-node src/test/programs/federal/snap/generateTestCases",
-    "lint": "./node_modules/.bin/eslint src --fix"
+    "deploy": "run -s build gh-pages",
+    "gh-pages": "gh-pages -d build/",
+    "generateSNAPTests": "babel-node src/test/programs/federal/snap/generateTestCases",
+    "lint:eslint": "eslint src",
+    "lint:prettier": "prettier --list-different '**/*.{md,json}'",
+    "fix:eslint": "eslint src --fix",
+    "fix:prettier": "prettier --write '**/*.{md,json}'"
   },
   "homepage": "https://codeforboston.github.io/cliff-effects",
   "devDependencies": {
     "babel-cli": "^6.26.0",
+    "babel-eslint": "^7.2.3",
     "babel-preset-env": "^1.6.1",
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
     "codecov": "^3.0.4",
+    "cross-env": "^5.2.0",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "enzyme-to-json": "^3.3.3",
     "eslint": "^4.19.1",
+    "eslint-config-react-app": "^2.1.0",
     "eslint-plugin-babel": "^5.1.0",
-    "eslint-plugin-jest": "^21.15.1",
-    "gh-pages": "^1.1.0",
+    "eslint-plugin-flowtype": "^2.50.0",
+    "eslint-plugin-import": "^2.13.0",
+    "eslint-plugin-jest": "^21.17.0",
+    "eslint-plugin-jsx-a11y": "^5.1.1",
+    "eslint-plugin-react": "^7.10.0",
+    "gh-pages": "^1.2.0",
+    "npm-run-all": "^4.1.3",
+    "prettier": "^1.13.7",
+    "react-scripts": "^1.1.4",
```

Comment:
I didn't like the idea of moving `react-scripts` to `devDependencies` given that create-react-app put it in `dependencies` by default, but it might be OK. I'm also confused about why apparently the change caused several other packages to be required here - maybe caused by the upgrade to `react-scripts` rather than the move? Could we try reverting the version and removing the extra packages? Unless we can figure out why this happened and make sure it's a good thing.

Reply:
I submit a commit to move react-scripts back to dependencies.